test/self_doc_golden.md

```diff
@@ -14,152 +14,25 @@ Generates documentation for exported skylark rule definitions in a target starla
-      <td>
-        <a href="https://bazel.build/docs/build-ref.html#labels">Label</a>; optional
-        <p>
-          The starlark file to generate documentation for.
-        </p>
-      </td>
-    </tr>
-    <tr id="stardoc-out">
-      <td><code>out</code></td>
-      <td>
-        <a href="https://bazel.build/docs/build-ref.html#labels">Label</a>; required
-        <p>
-          The (markdown) file to which documentation will be output.
-        </p>
-      </td>
-    </tr>
-    <tr id="stardoc-provider_template">
-      <td><code>provider_template</code></td>
-      <td>
-        <a href="https://bazel.build/docs/build-ref.html#labels">Label</a>; optional
-        <p>
-          The input file template for providers generated in documentation.
-        </p>
-      </td>
-    </tr>
-    <tr id="stardoc-renderer">
-      <td><code>renderer</code></td>
-      <td>
-        <a href="https://bazel.build/docs/build-ref.html#labels">Label</a>; optional
-        <p>
-          The location of the renderer tool.
-        </p>
-      </td>
-    </tr>
-    <tr id="stardoc-rule_template">
-      <td><code>rule_template</code></td>
-      <td>
-        <a href="https://bazel.build/docs/build-ref.html#labels">Label</a>; optional
-        <p>
-          The input file template for rules generated in documentation.
-        </p>
-      </td>
-    </tr>
-    <tr id="stardoc-semantic_flags">
-      <td><code>semantic_flags</code></td>
-      <td>
-        List of strings; optional
-        <p>
-          A list of canonical flags to affect Starlark semantics for the Starlark interpretter
-during documentation generation. This should only be used to maintain compatibility with
-non-default semantic flags required to use the given Starlark symbols.
-<br><br>For example, if <code>//foo:bar.bzl</code> does not build except when a user would specify
-<code>--incompatible_foo_semantic=false</code>, then this attribute should contain
-"--incompatible_foo_semantic=false".
-        </p>
-      </td>
-    </tr>
-    <tr id="stardoc-stardoc">
-      <td><code>stardoc</code></td>
-      <td>
-        <a href="https://bazel.build/docs/build-ref.html#labels">Label</a>; optional
-        <p>
-          The location of the stardoc tool.
-        </p>
-      </td>
-    </tr>
-    <tr id="stardoc-symbol_names">
-      <td><code>symbol_names</code></td>
-      <td>
-        List of strings; optional
-        <p>
-          A list of symbol names to generate documentation for. These should correspond to
-the names of rule definitions in the input file. If this list is empty, then
-documentation for all exported rule definitions will be generated.
-        </p>
-      </td>
-    </tr>
-  </tbody>
-</table>
+**ATTRIBUTES**
+
+
+| Name  | Description | Type | Mandatory |
+| :-------------: | :-------------: | :-------------: | :-------------: |
+| name |  A unique name for this target.   | <a href="https://bazel.build/docs/build-ref.html#name">Name</a> | required |
```

Comment:
Can we use markdown links instead? `[Name](https://bazel.build/docs/build-ref.html#name)`

Reply:
Good idea. Could we do this as part of another CL?

For better or worse, the use of <a> tags is in MarkdownUtil, so either I'll need to modify the templates to not use these functions, or, more ideally, I'd modify / extend MarkdownUtil and cut new jars. Regardless, it's a little extra work.